src/components/Badge/Badge.style.ts

```diff
@@ -0,0 +1,24 @@
+import styled from '@emotion/styled';
+import theme from '@styles/theme';
+
+interface Props {
+  color: string;
+  width: string;
+  height: string;
+  fontSize: string;
+  fontColor: string;
+  borderRadius: string;
+}
+
+export const Container = styled.div<Props>`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 1rem;
+  width: ${({ width }) => width};
+  height: ${({ height }) => height};
+  font-size: ${({ fontSize }) => fontSize};
+  border-radius: ${({ borderRadius }) => borderRadius};
+  color: ${({ fontColor }) => (fontColor === 'primary' ? `${theme.color.background}` : fontColor)};
+  background-color: ${({ color }) => (color === 'primary' ? `${theme.color.green200}` : color)};
```

Comment:
이 부분 창민님이 말씀하셨던 것처럼 상단에 있는 
```javascript
import theme from '@styles/theme'
```
제거하고 아래와 같이 변경하는 건 어떨까요??
```suggestion
  color: ${({ fontColor, theme }) => (fontColor === 'primary' ? `${theme.color.background}` : fontColor)};
  background-color: ${({ color, theme }) => (color === 'primary' ? `${theme.color.green200}` : color)};
```

Reply:
네네!!